reana_client/cli/workflow.py

```diff
@@ -820,6 +821,52 @@ def print_color_diff(lines):
             err=True)
 
 
+@click.command(
+    'open',
+    help='Open an interactive session inside the workflow workspace')
+@click.argument(
+    'workflow',
+    default=os.environ.get('REANA_WORKON', None),
+    callback=workflow_uuid_or_name)
+@click.option(
+    '-i',
+    '--image',
+    help="Image to use in the interactive session, by default it is"
+         "``jupyter/scipy-notebook``.")
+@click.option(
+    '-p',
+    '--port',
+    type=int,
+    help="Port exposed by the service run inside the container spawned"
```

Comment:
space missing. 

>   -p, --port INTEGER        Port exposed by the service run inside the
>                             container spawnedwith ``image``, by default

Maybe it is a bit too techincal help description for user. What about something like this?

> Specify a port for the interactive session.

What do you think?

Reply:
Definitely 👍 changing ...